Lib/test/test_poll.py

```diff
@@ -130,7 +130,7 @@ def test_poll2(self):
         p = proc.stdout
         pollster = select.poll()
         pollster.register( p, select.POLLIN )
-        for tout in (0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
+        for tout in (-1000, 0, 1000, 2000, 4000, 8000, 16000) + (-1,)*10:
```

Comment:
This is not directly related to this ticket, but I realized that `None` is not tested in `test_poll`. Could you also add `None` while we are here?